pkg/sql/sem/tree/copy.go

```diff
@@ -30,4 +31,8 @@ func (node *CopyFrom) Format(ctx *FmtCtx) {
 	if node.Stdin {
 		ctx.WriteString("STDIN")
 	}
+	if node.Options != nil {
```

Comment:
You need to add one of these to sql/parser/parse_test.go which will verify that we can parse, print, and pretty print these correctly.